packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.tsx

```diff
@@ -0,0 +1,53 @@
+import {Component, Element, h, getAssetPath, Host} from '@stencil/core';
```

Comment:
Main file

Reply:
Let's say I want to display my own e-commerce icons. How can we adapt the `atomic-result-icon` component to allow this?

Or should we have a separate component? And we would rename the current component to something like `atomic-result-sf-icon` or `atomic-result-magic-icon`.

Reply:
You could add the e-commerce icons in the assets, and if would work the same without additional work.